tools/flight_recorder/components/builder.py

```diff
@@ -284,11 +300,17 @@ def build_collectives(
                     check_result, input_numel, output_numel = check_size_alltoall(
                         alltoall_cases
                     )
+                    # We don't log the input/output sizes for alltoall so we don't consider the size mismatch as an error for now.
+                    check_result = True
                     if not check_result:
+                        # When we see errors in all_to_all, it's hard to tell which rank is the source of the error.
                         mismatch[pg_name] += 1
                         print(
-                            f"Input/output mismatch in the collective for group {pg_name}:{desc} collective {profiling_name}",
-                            f"input_numel {input_numel} output_numel{output_numel}",
+                            f"Input/output mismatch in the collective {record_id} ",
+                            f"for group {pg_name}:{desc} collective {profiling_name} ",
+                            f"input_numel {input_numel} output_numel {output_numel} ",
+                            f"{input_sizes} {output_sizes} {len(expected_ranks)} {collective_state} ",
+                            f"\nCollective stack traces: \n{colletive_frames}",
```

Comment:
Typo here = unless it needs to be fixed elsewhere?
```suggestion
                            f"\nCollective stack traces: \n{collective_frames}",
```